include/glow/Graph/Graph.h

```diff
@@ -478,6 +478,11 @@ class Function final : public Named {
   /// Result type will be implicitly set based on the \p input type.
   TanhNode *createTanh(llvm::StringRef name, NodeValue input);
 
+  /// Create an Exp  node with \p name, which calculates element-wise
+  /// exponential of \p input, with output type \p outTy.
+  ExpNode *createExp(llvm::StringRef name, NodeValue input,
```

Comment:
@rdzhabarov  Thanks for a quick review.  Looking at the current state of tests, I get an impression that the tests are added only for ops where there are attributes involved, or some special input/output scenarios. There is no test for log, tanh, sigmoid, which have similar template. So, I am not clear what's the criteria?

Reply:
> Looking at the current state of tests, I get an impression that the tests are added only for ops where there are attributes involved

might be a coincidence. 

Output type is often used during the quantization of the graph so you can set different scale and offset for input/output.

Reply:
Regarding the outTy in function signature: 
I tried to model this upon Log which uses the same template.

Regarding the onnxImporterTest/caffe2ImporterTest:
I am not convinced if we really need a test for simple operators like log/exp/tanH etc. (none of them exist as of now), where there is nothing really to test in terms of parsing just the operator name. As far as functionality is concerned it's already being tested in the OperatorTest. Please let me know if there exists a simple test builder for such kind of tests.

Reply:
missing tests for log/exp/tanH is not a feature but a bug :). I think it makes sense to test loading logic otherwise how do you know that you load it properly ;).

Reply:
> I tried to model this upon Log which uses the same template.

we can add this when it's needed. I'd remove now.

Reply:
I think we need a generic function for testing of loading Unary OPs. Strange it was never added when so many unary operators already exist. I will add that so that it benefits testing of other unary ops as well in future.